### DISCUSSION TITLE
Question about adjusting parameters

### DISCUSSION THREAD

**Discussion by Pedro-Floriano** [mboquien/cigale#49] [Q&A]
Hello everyone, I've been studying how to best use cigale for my research, and for that I started running the code with the photometry of the standard star HR1544, before hitting `$ pcigale check` I've been editing the parameters by hand in the pcigale.ini document. I'll link the results I'm getting.
![WhatsApp Image 2023-03-20 at 11 11 29 PM](https://user-images.githubusercontent.com/113158126/232178092-4a9a8975-cc43-496e-9dec-31dec26f3457.jpeg)
You can see that the fit (bottom) isn't really good compared with the literature (top). My question is, is there a way to ask cigale to adjust the parameters by itself, in a way to produce a more accurate fit?
I'm also linking the data I'm fitting and the pcigale.ini that I used.
[hr1544.txt](https://github.com/mboquien/cigale/files/11238083/hr1544.txt)
[pcigale_ini.txt](https://github.com/mboquien/cigale/files/11238085/pcigale_ini.txt)

Thank you, in advance.

**Comment by shivathanay-p** [comment]
You can have CIGALE build many models for different combinations of parameters by simply choosing multiple parameter values (comma-separated) for each parameter in the pcigale.ini file. The possible parameter values are listed in the comments of the pcigale.ini file.

**Reply by 516Athena** [reply]
hi, if I. choose different parameter values,CIGALE will estimate which values is. the. best. ?

**Reply by mboquien** [reply]
A grid should always be defined, sampling the relevant parameters for the specific science case. Then the columns starting with `best` in the results file give the set of parameters corresponding to the lowest χ².
